frontend/components/Domain/Recipe/RecipeCardSection.vue

```diff
@@ -261,8 +269,12 @@ export default defineComponent({
 
     async function initRecipes() {
       page.value = 1;
+      hasMore.value = true;
+
+      // we double-up the first call to avoid a bug with large screens that render
+      // the entire first page without scrolling, preventing additional loading
       const newRecipes = await fetchRecipes(2);
```

Comment:
Not part of the commit but since you added an explanation i thought you had your problems determining why. I think this makes it a bit more clear what is happening here by not having a "random" 2 as value. 

```suggestion
      const newRecipes = await fetchRecipes(page.value + 1);
```

Reply:
I think I had moved the comment which was just in the wrong place, but I agree this is better anyway